src/Microsoft.AspNet.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -34,9 +37,8 @@ public DateHeaderValueManager()
             : this(
                   systemClock: new SystemClock(),
                   timeWithoutRequestsUntilIdle: TimeSpan.FromSeconds(10),
-                  timerInterval: TimeSpan.FromSeconds(1))
+                  timerInterval: TimeSpan.FromMilliseconds(200))
```

Comment:
Not sure running 5 times as often is necessary? As it doesn't need to be highly accurate (reasons given in issue)